### ISSUE TITLE
0995 - 508 audit defect 8 - Fix the VaAccordion expand/collapse all button aria-label

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#109146]
## Issue
- This ticket is for tracking fixes for Defect 8 from the recent 508 audit on the 0995. It will require changes to the design system `VAAccordion` component. Need to do some digging and see why this hasn't been updated before. I think I've seen this suggestion from the 508 office before, and talk about the accordion toggle button in slack.

> WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.
> 
> ~On the File A Supplemental Claim | Veterans Affairs screen (We've prefilled some of your information), when tab is used to move the focus to Start your claim, “Start your claim same page link” is read—this link navigates to another page. (New: 04/04/2025)~ - FIXED
> 
> On the File A Supplemental Claim | Veterans Affairs screen (Step5 of 5Review application), under Step5 of 5Review application, the name of the button cannot be understood in relation to the state. For example, “Expand all accordions Button collapsed” is read. One possibility is to change the name, “Expand all accordions” or “Collapse all accordions”, without indicating the state. Another possibility is to keep the name the same, “all accordions Button collapsed” or “all accordions Button expanded”, and change the state—this is the convention used throughout the product. (New: 04/04/2025)

## Acceptance Criteria
- [ ] The accordion button announces more clearly when used with assistive technology.

## Resources
[[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106900#top)
[[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms - Defect 8 | Critical #107049](https://github.com/department-of-veterans-affairs/va.gov-team/issues/107049)

**Comment by bacitracin** [comment]
Filed an issue for this with the VA Design System team https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4531

**Comment by bacitracin** [comment]
Component has been updated - https://github.com/department-of-veterans-affairs/component-library/pull/1772

**Comment by bacitracin** [comment]
The new expand and collapse button is live, closing this ticket. 
<img width="1416" height="1076" alt="Image" src="https://github.com/user-attachments/assets/7978aa4d-beb4-4dae-bea9-9172f2186533" />
